frontend/static/js/App.js

```diff
@@ -0,0 +1,218 @@
+                        className: "fas fa-list",
+                        title: "List",
+                    },
+                    {
+                        name: "url",
+                        action: SimpleMDE.drawLink,
+                        className: "fas fa-link",
+                        title: "Insert URL",
+                    },
+                    {
+                        name: "image",
+                        action: SimpleMDE.drawImage,
+                        className: "fas fa-image",
+                        title: "Insert an image",
+                    },
+                    {
+                        name: "code",
+                        action: SimpleMDE.toggleCodeBlock,
+                        className: "fas fa-code",
+                        title: "Insert code",
+                    },
+                ],
+                spellChecker: false,
+                forceSync: true,
+                tabSize: 4,
+            });
+            allEditors.push(editor);
+            inlineAttachment.editors.codemirror4.attach(editor.codemirror, imageUploadOptions);
+        }
+
+        const invisibleMarkdownEditors = document.querySelectorAll(".markdown-editor-invisible");
+        for (let i = 0; i < invisibleMarkdownEditors.length; i++) {
+            let editor = new SimpleMDE({
+                element: invisibleMarkdownEditors[i],
+                autoDownloadFontAwesome: false,
+                toolbar: false,
+                status: false,
+                spellChecker: false,
+                forceSync: true,
+                tabSize: 4,
+            });
+            allEditors.push(editor);
+            inlineAttachment.editors.codemirror4.attach(editor.codemirror, imageUploadOptions);
+        }
+        return allEditors;
+    },
+    addTargetBlankToExternalLinks() {
+        let internal = location.host.replace("www.", "");
+        internal = new RegExp(internal, "i");
+
+        let a = document.getElementsByTagName("a");
+        for (let i = 0; i < a.length; i++) {
+            let href = a[i].host;
+            if (!internal.test(href)) {
+                a[i].setAttribute("target", "_blank");
+            }
+        }
+    },
+    initializeImageZoom() {
+        Lightense(document.querySelectorAll(".text-body figure img"), {
+            time: 100,
+            padding: 40,
+            offset: 40,
+            keyboard: true,
+            cubicBezier: "cubic-bezier(.2, 0, .1, 1)",
+            background: "rgba(0, 0, 0, .4)",
+            zIndex: 2147483647,
+        });
+    },
+    bindCommentsHotkey() {
+        const commentForm  = document.querySelector('.comment-form-form');
+        if (this.isMobile() || !commentForm) { return; }
+
+        commentForm.addEventListener('keydown', (event) => this.handleCommentHotkey(event, commentForm));
+    },
+    handleCommentHotkey(event, controlElement) {
+        if ((event.ctrlKey || event.metaKey) && event.keyCode === 13) {
```

Comment:
keyCode is currently deprecated: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode
You could try using `key` instead ([browser support](https://caniuse.com/#feat=keyboardevent-key)): 
```suggestion
        if ((event.ctrlKey || event.metaKey) && event.key === "Enter") {
```